src/helics/application_api/ValueFederateManager.hpp

```diff
@@ -161,10 +161,6 @@ class ValueFederateManager
     /** get an option values for a publication*/
     bool getPublicationOption (const Publication &pub, int32_t option) const;
 
-    /** register a callback function to call when any subscribed value is updated
-    @details there can only be one generic callback
-    @param[in] callback the function to call
-    */
```

Comment:
why was this deleted?

Reply:
It got caught up in an overzealous delete of some functions which had been placed directly above it. I've restored it on local and committed it. I will push that up with the other fixes when they're ready.